src/usecase/tests/order_ucase_test.go

```diff
@@ -0,0 +1,125 @@
+package tests
+
+import (
+	"github.com/Aiscom-LLC/meals-api/src/delivery"
+	"github.com/Aiscom-LLC/meals-api/src/delivery/middleware"
+	"github.com/Aiscom-LLC/meals-api/src/repository"
+	"github.com/appleboy/gofight/v2"
+	"github.com/buger/jsonparser"
+	"github.com/go-playground/assert/v2"
+	uuid "github.com/satori/go.uuid"
+	"net/http"
+	"testing"
+)
+
+func TestAddOrder(t *testing.T) {
+	r := gofight.New()
+
+	type Dish struct {
+		Amount int    `json:"amount"`
+		DishId string `json:"dishId"`
+	}
+	type Order struct {
+		Comment string `json:"comment"`
+		Items   []Dish `json:"items"`
+	}
+
+	userRepo := repository.NewUserRepo()
+	cateringRepo := repository.NewCateringRepo()
+	categoryRepo := repository.NewCategoryRepo()
+	cateringResult, _ := cateringRepo.GetByKey("name", "Twiist")
+	cateringID := cateringResult.ID.String()
+	categoryResult, _ := categoryRepo.GetByKey("name", "гарнир", cateringID)
+	categoryID := categoryResult.ID.String()
+	userResult, _ := userRepo.GetByKey("email", "admin@meals.com")
+	dishRepo := repository.NewDishRepo()
+	dishResult, _, _ := dishRepo.GetByKey("name", "доширак", cateringID, categoryID)
+	dishID := dishResult.ID.String()
+	newUser, _ := userRepo.GetByKey("email", "user1@meals.com")
+	userID := newUser.ID.String()
+	jwt, _, _ := middleware.Passport().TokenGenerator(&middleware.UserID{ID: userResult.ID.String()})
+	var test Order
+	var test1 Dish
+	test1.Amount = 1
+	test1.DishId = dishID
+	test.Comment = "cool comment"
+	test.Items = append(test.Items, test1)
```

Comment:
let's change naming to more appropriate one e.g
`var dish Dish`
`var order Order`
`dish.Amount = 1`
`dish.ID = someID`

... etc.